File: benchmarks/xlang/src/strbuild.js
const n = 2000000;
const t0 = performance.now();
let s = "";
for (let i = 0; i < n; i++) s += String(i % 1000);
console.log(`CHECK ${s.length}`);
console.log(`MS ${Math.round(performance.now() - t0)}`);

File: benchmarks/xlang/src/strbuild.rs
fn main() {
    let n = 2000000;
    let t0 = std::time::Instant::now();
    let mut s = String::new();
    for i in 0..n {
        s.push_str(&(i % 1000).to_string());
    }
    println!("CHECK {}", s.len());
    println!("MS {}", t0.elapsed().as_millis());
}
